utils/openlittermap_downloader.py

```diff
@@ -0,0 +1,55 @@
+import urllib.request
+import json
+import requests
+import os
+
+
+def create_json(path, period):
+    data = requests.get(f'https://openlittermap.com/global-data?date={period}')
+    if data.ok:
+        geojson = json.loads(data.text)['geojson']
+    else:
+        raise ValueError('Got invalid response from the server.')
+    files = {os.path.splitext(file)[0] for file in os.listdir(path) if os.path.isfile(file)}
+    features = {feature['properties']['photo_id']: feature['properties'] for feature in geojson['features'] if str(feature['properties']['photo_id']) in files}
+
+    with open(os.path.join(path, '..', 'jsondata.json'), 'w') as file:
+        json.dump(features, file)
+
+
+def download_from_time_period(period:str):
+    """
+    :param period: 'one-year', 'one-week', 'one-month', 'today'
+    """
+    data = requests.get(f'https://openlittermap.com/global-data?date={period}')
+    
+    if data.ok:
+        geojson = json.loads(data.text)['geojson']
+    else:
+        raise ValueError('Got invalid response from the server.')
+    
+    jsondata = dict()#{feature['properties']['photo_id']: feature['properties'] for feature in geojson['features']}
+
+
+    i = 0
+    for feature in geojson['features']:
+        _, file_extension = os.path.splitext(feature['properties']['filename'])
+        try:
+            file_path = '/dih4/dih4_2/wimlds/zklawikowska/openlittermap/images/%s%s' % (feature['properties']['photo_id'], file_extension)
```

Comment:
file_path is hardcoded - maybe it will be better to pass it as argument to the function